libs/features/icons/icons.js

```diff
@@ -55,13 +65,22 @@ export default async function loadIcons(icons, config) {
   const iconSVGs = await fetchIcons(config);
   if (!iconSVGs) return;
   icons.forEach(async (icon) => {
-    const { classList } = icon;
-    if (classList.contains('icon-tooltip')) decorateToolTip(icon);
-    const iconName = icon.classList[1].replace('icon-', '');
+    const iconNameInitial = icon.classList[1].replace('icon-', '');
+    let iconName;
+
+    if (iconNameInitial === 'tooltip') {
+      iconName = 'info';
+    } else if (iconNameInitial.includes('tooltip-')) {
+      iconName = iconNameInitial.replace(/tooltip-/, '');
+    } else {
+      iconName = iconNameInitial;
+    }
+
+    if (checkIfIconIsTooltip(icon)) decorateToolTip(icon, iconName);
```

Comment:
I'd put the `if` condition within the function where its used as that encapsulates the whole logic a bit better